synapse/lib/schemas.py

```diff
@@ -1064,6 +1263,7 @@
             'additionalItems': True,
             'required': ('name',),
         },
+        'jsonschema': s_msgpack.deepcopy(_jsonSchemaSchema)
```

Comment:
I don't understand why this is here. I don't see it being used anywhere?

If you do need it, I think you can make this line something like this instead:

```suggestion
        'jsonschema': {'$ref': 'http://json-schema.org/draft-07/schema#'},
```

And then maybe delete the schema copy above?